mods/common/hotkeys/game.yaml

```diff
@@ -18,6 +18,10 @@ SelectUnitsByType: W
 	Description: Select units by type
 	Types: World, Player, Spectator
 
+CycleHarvesters: N
```

Comment:
I bet that putting this right next to the dangerously confusing `M` key on the keyboard is going to cause trouble. I'd rather we fix this by adding a visual indicator for mute than the change the hotkey.

Reply:
Agreed. I often press the mute key by accident. Filed #17549